airflow/www/static/js/api/useClearRun.ts

```diff
@@ -17,30 +17,30 @@
  * under the License.
  */
 
-import axios from 'axios';
+import axios, { AxiosResponse } from 'axios';
 import { useMutation, useQueryClient } from 'react-query';
 import { getMetaValue } from '../utils';
 import { useAutoRefresh } from '../context/autorefresh';
 import useErrorToast from '../utils/useErrorToast';
 
-const csrfToken = getMetaValue('csrf_token');
-const clearRunUrl = getMetaValue('dagrun_clear_url');
+const csrfToken = getMetaValue('csrf_token') || '';
+const clearRunUrl = getMetaValue('dagrun_clear_url') || '';
 
-export default function useClearRun(dagId, runId) {
+export default function useClearRun(dagId: string, runId: string) {
   const queryClient = useQueryClient();
   const errorToast = useErrorToast();
   const { startRefresh } = useAutoRefresh();
   return useMutation(
     ['dagRunClear', dagId, runId],
-    ({ confirmed = false }) => {
+    ({ confirmed = false }: { confirmed: boolean }) => {
       const params = new URLSearchParams({
         csrf_token: csrfToken,
-        confirmed,
+        confirmed: confirmed.toString(),
```

Comment:
I wonder if it's worth it to make a URLSearchParams wrapper to convert everything to strings automatically.

Reply:
Done